dataeng/resources/enrollment-validation-events.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+END_DATE=$(date +%Y-%m-%d -d "$TO_DATE")
+START_DATE=$(date +%Y-%m-%d -d "$FROM_DATE")
+INTERVAL=$START_DATE-$END_DATE
+
+if [ -z "$NUM_REDUCE_TASKS" ]; then
+    NUM_REDUCE_TASKS=$(( $NUM_TASK_CAPACITY + 1 ))
+fi
+env|sort
+
+${WORKSPACE}/analytics-configuration/automation/run-automated-task.sh \
+  EnrollmentValidationWorkflow --local-scheduler \
+  --output-root $OUTPUT_ROOT/$INTERVAL \
+  --interval $INTERVAL \
+  --credentials $CREDENTIALS \
+  --n-reduce-tasks $NUM_REDUCE_TASKS
+
+pip install awscli
+aws s3 ls $OUTPUT_ROOT/$INTERVAL/ | grep "synthetic_enroll"  || true
+
+MAX_FILE_SIZE=`aws s3 ls $OUTPUT_ROOT/$INTERVAL/ | cut -b20-30 | grep -v PRE | sort -nr | head -1`
+if [ "$MAX_FILE_SIZE" -gt "$FILE_THRESHOLD" ]; then echo "OVER" "THRESHOLD"; fi
```

Comment:
This is going to break on edge, with or without a post-build step, or in any case where there are no synthetic events.  Maybe we can put an if-then-else here to handle if MAX_FILE_SIZE="" and FILE_THRESHOLD>="0" then do nothing.